client/session_bootstrap_blocks_test.go

```diff
@@ -1104,7 +1110,8 @@ func (m *fetchBlocksReqMatcher) Matches(x interface{}) bool {
 	}
 
 	for i := range params {
-		if params[i].id != req.Elements[i].ID {
+		id := ts.BinaryID(req.Elements[i].ID)
+		if params[i].id.Hash() != id.Hash() {
```

Comment:
Again, not sure about hash comparison instead of deep equal (which is what default string is)